src/SPC/command/DownloadCommand.php

```diff
@@ -59,105 +59,110 @@ public function initialize(InputInterface $input, OutputInterface $output): void
      */
     public function handle(): int
     {
-        // 删除旧资源
-        if ($this->getOption('clean')) {
-            logger()->warning('You are doing some operations that not recoverable: removing directories below');
-            logger()->warning(SOURCE_PATH);
-            logger()->warning(DOWNLOAD_PATH);
-            logger()->warning(BUILD_ROOT_PATH);
-            logger()->alert('I will remove these dir after 5 seconds !');
-            sleep(5);
-            if (PHP_OS_FAMILY === 'Windows') {
-                f_passthru('rmdir /s /q ' . SOURCE_PATH);
-                f_passthru('rmdir /s /q ' . DOWNLOAD_PATH);
-                f_passthru('rmdir /s /q ' . BUILD_ROOT_PATH);
-            } else {
-                f_passthru('rm -rf ' . SOURCE_PATH . '/*');
-                f_passthru('rm -rf ' . DOWNLOAD_PATH . '/*');
-                f_passthru('rm -rf ' . BUILD_ROOT_PATH . '/*');
+        try {
+            if ($this->getOption('clean')) {
+                logger()->warning('You are doing some operations that not recoverable: removing directories below');
+                logger()->warning(SOURCE_PATH);
+                logger()->warning(DOWNLOAD_PATH);
+                logger()->warning(BUILD_ROOT_PATH);
+                logger()->alert('I will remove these dir after 5 seconds !');
+                sleep(5);
+                if (PHP_OS_FAMILY === 'Windows') {
+                    f_passthru('rmdir /s /q ' . SOURCE_PATH);
+                    f_passthru('rmdir /s /q ' . DOWNLOAD_PATH);
+                    f_passthru('rmdir /s /q ' . BUILD_ROOT_PATH);
+                } else {
+                    f_passthru('rm -rf ' . SOURCE_PATH . '/*');
+                    f_passthru('rm -rf ' . DOWNLOAD_PATH . '/*');
+                    f_passthru('rm -rf ' . BUILD_ROOT_PATH . '/*');
+                }
+                return static::FAILURE;
             }
-            return static::FAILURE;
-        }
 
-        // --from-zip
-        if ($path = $this->getOption('from-zip')) {
-            return $this->downloadFromZip($path);
-        }
+            // --from-zip
+            if ($path = $this->getOption('from-zip')) {
+                return $this->downloadFromZip($path);
+            }
 
-        // Define PHP major version
-        $ver = $this->php_major_ver = $this->getOption('with-php') ?? '8.1';
-        define('SPC_BUILD_PHP_VERSION', $ver);
-        preg_match('/^\d+\.\d+$/', $ver, $matches);
-        if (!$matches) {
-            logger()->error("bad version arg: {$ver}, x.y required!");
-            return static::FAILURE;
-        }
+            // Define PHP major version
+            $ver = $this->php_major_ver = $this->getOption('with-php') ?? '8.1';
```

Comment:
8.2 please!

Reply:
The default value for the `--with-php` option is already 8.2, of course, this was just a missed update.

Reply:
ah yeah can just remove the ?? then